### PR TITLE
Update special issue manager to work for all RRD disabilities

### PR DESCRIPTION
**Pull request by leikkisa**
## Description of change
This PR adds the RRD special issue to single-issue asthma claims for increase

Attaching the “RRD” special issue routes the claim to the Boise RO so that it can be reviewed by RSVRs piloting RRD.

## Original issue(s)
https://www.notion.so/Submit-asthma-PDF-to-EVSS-316fa959aca24b0b8163264cd58aeb6a

## Things to know about this PR
* Feature flag: `rrd_asthma_release_pdf` (sets `release_pdf?` to true for asthma)
  * Added in #9613

Other related previous PRs:
- #9518 
- #9580

Testing
- Unit test added to check that the special issue is added for asthma claims

We are not aware of how to check NWQ routing in staging.

These are additional testing planned to check work before the NWQ routing:
- RRD Asthma PDFs are currently being uploaded to AWS to be manually checked
- Testing a single issue asthma claim for increase generates PDF in staging.

### CODE REVIEW

File: app/services/rapid_ready_for_decision/rrd_special_issue_manager.rb [hunk 1]
```diff
@@ -21,11 +21,13 @@ def add_special_issue
 
     private
 
-    HYPERTENSION_DISABILITY = RapidReadyForDecision::Constants::DISABILITIES[:hypertension]
+    RRD_DIAGNOSTIC_CODES = RapidReadyForDecision::Constants::DISABILITIES_BY_CODE.keys
```
Comment (yoomlam) [inline app/services/rapid_ready_for_decision/rrd_special_issue_manager.rb:24]: Interesting that the linter didn't ask for a `.freeze` for this constant.

File: spec/services/rapid_ready_for_decision/rrd_special_issue_manager_spec.rb [hunk 2]
```diff
@@ -12,7 +12,7 @@ def form526_hash(form)
 
   def filter_disabilities(form)
```
Comment (yoomlam) [inline spec/services/rapid_ready_for_decision/rrd_special_issue_manager_spec.rb:13]: Suggestion: use a more descriptive method name:
```suggestion
  def supported_disabilities(form)
```

File: app/services/rapid_ready_for_decision/rrd_special_issue_manager.rb (hunk 1 above)
Comment (leikkisa) [reply app/services/rapid_ready_for_decision/rrd_special_issue_manager.rb:24]: Will go ahead and add that!

Comment (allthesignals) [reply app/services/rapid_ready_for_decision/rrd_special_issue_manager.rb:24]: Yeahhh interesting. In my case it gave a syntax error for 
```ruby
ASTHMA_KEYWORDS = RapidReadyForDecision::Constants::DISABILITIES[:asthma][:keywords] # syntax error: dynamic constant assignment
```

Maybe it's considered "dynamic" if it's nested keys (`DISABILITIES[:asthma][:keywords]`)

**Review by yoomlam — APPROVED** [review]

**Review by RachalCassity — APPROVED** [review]

**Pull request merged**